web-admin/src/features/ai/MCPConfigSection.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import { ADMIN_URL } from "@rilldata/web-admin/client/http-client";
+  import CodeBlock from "@rilldata/web-common/components/code-block/CodeBlock.svelte";
+
+  export let organization: string;
+  export let project: string;
+  export let isPublic: boolean;
+
+  // Construct the API URL for the MCP server
+  $: apiUrl = `${ADMIN_URL}/v1/organizations/${organization}/projects/${project}/runtime/mcp/sse`;
```

Comment:
Do we want to do a replace for `admin.rilldata` with `api.rilldata`. The recommended urls when proxying are https://api.rilldata.com/.... instead of https://admin.rilldata.com/....

Reply:
Yeah, what is the right logic here? Is it just a find/replace? What is recommended for custom domains?

Reply:
`api.rilldata.com` is the best one to use for user-facing endpoints (eventually we should also switch away from `admin.rilldata.com` internally, but that'll require some extra QA). We don't support MCP on custom domains at the moment.

Reply:
Thanks, updated